mne/datasets/limo/limo.py

```diff
@@ -0,0 +1,253 @@
+    fname_eeg = op.join(limo_path, subj, 'Yr.mat')
+    data = scipy.io.loadmat(fname_eeg)
+
+    # -- 2) get epochs information from structure
+    # sampling rate
+    sfreq = data_info['LIMO']['data'][0][0][0][0]['sampling_rate'][0][0]
+    # tmin and tmax
+    tmin = data_info['LIMO']['data'][0][0][0][0]['start'][0][0]
+    # number of epochs per condition
+    design = data_info['LIMO']['design'][0][0]['X'][0][0]
+
+    # create events matrix
+    events = np.transpose(np.array([list(range(len(design))),
+                                    np.zeros(len(design)),
+                                    design[:, 1]]).astype(int))
+    # event ids, such that Face B == 1
+    event_id = {'Face A': 0, 'Face B': 1}
+
+    # -- 3) extract channel labels from LIMO structure
+    # get individual labels
+    labels = data_info['LIMO']['data'][0][0][0][0]['chanlocs']['labels']
+    labels = [ll[0] for ll in labels[0]]
+    # get montage
+    montage = mne.channels.read_montage('biosemi128')
+    # add external electrodes (e.g., eogs)
+    ch_names = montage.ch_names[:-3] + ['EXG1', 'EXG2', 'EXG3', 'EXG4']
+    # match individual labels to labels in montage
+    found_inds = [ii for ii, ch_name in enumerate(ch_names) if ch_name in labels]
+    missing_chans = [ch for ch in ch_names if ch not in labels]
+    assert labels == [ch_names[ii] for ii in found_inds]
+
+    # -- 4) extract data from subjects Yr structure
+    # data is stored as channels x time points x epochs
+    # data['Yr'].shape  # <-- see here
+    # transpose to epochs x channels time points
+    obs_data = np.transpose(data['Yr'], (2, 0, 1))
+
+    # initialize data in expected order
+    all_data = np.empty((obs_data.shape[0],
+                         len(ch_names),
+                         obs_data.shape[2]))
+    # copy over the non-missing data.
+    for source, target in enumerate(found_inds):
+        # avoid copy when fancy indexing.
+        all_data[:, target, :] = obs_data[:, source, :]
+    # data to V (to match MNE)
+    obs_data = all_data / 1e6
+
+    # create list containing channel types
+    types = ["eog" if ch.startswith("EXG") else "eeg" for ch in ch_names]
+
+    # -- 5) Create custom info for mne epochs structure
+    # create info
+    info = mne.create_info(ch_names=ch_names, ch_types=types, sfreq=sfreq)
+
+    # get faces and noise variables from design matrix
+    faces = ['A' if list(events[:, 2])[i] == 0 else 'B' for i in list(events[:, 2])]
+    noise = list(design[:, 2])
+    # create epochs metadata
+    metadata = {'Face': faces,
+                'Noise': noise}
+    metadata = pd.DataFrame(metadata)
+
+    # -- 6) Create custom epochs array
+    epochs = mne.EpochsArray(obs_data, info, events, tmin, event_id, metadata=metadata)
+    epochs.set_montage(montage=montage)
+    epochs.info['bads'] = missing_chans  # missing channels are marked as bad.
+
+    # -- 7) interpolate missing channels
+    if interpolate is True and montage is not None:
```

Comment:
`if interpolate and montage:` might be enough. *Might*.

Reply:
I change this to ``if interpolate:``, since im forcing the setting of the montage above.